src/utils/client.js

```diff
@@ -1,4 +1,4 @@
-import 'isomorphic-fetch';
+import 'unfetch/polyfill';
```

Comment:
A ponyfill (at least in unfetch's case) just means it doesn't get put as a global on `window`. All users will still use native fetch if possible. So there's no real downside.